### PR TITLE
Minor improvements and TODO resolution

### PR DESCRIPTION
**Pull request by stefanodellosa-personal**

### CODE REVIEW

File: src/domain/configure_ftp_cameras.py [hunk 1]
```diff
@@ -6,10 +6,9 @@
 from validators import ipv4
 
 from PySide6.QtWidgets import (QDialog, QDialogButtonBox, QFileDialog, QLineEdit, QRadioButton, QLabel, QGridLayout,
-                               QScrollArea, QWidget, QVBoxLayout, QGridLayout)
+                               QScrollArea, QWidget, QGridLayout)
 from PySide6.QtGui import QIcon
 from PySide6.QtCore import Qt
```
Comment (cdimauro) [inline src/domain/configure_ftp_cameras.py:11]: import logging
import os

import keyring
from validators import ipv4

from PySide6.QtCore import Qt
from PySide6.QtGui import QIcon
from PySide6.QtWidgets import (QDialog, QDialogButtonBox, QFileDialog, QGridLayout, QLabel, QLineEdit, QRadioButton, QScrollArea, QWidget)

from src.domain.camera import Camera, cameras
from src.domain.ftp_camera import FTPCamera
from src.domain.ftps_server import FTPsServer
from src.domain.qtextedit_logger import QTextEditLogger
from src.ui.ui_configure_ftp_cameras import Ui_DialogFTPCameras

File: src/domain/configure_ftp_cameras.py [hunk 2]
```diff
@@ -107,23 +106,17 @@ def list_ftp_cameras_in_tab(self):
     def accept_and_close(self):
         """When Ok is clicked, save FTP config info before closing."""
 
+        # If server object exists needs to be closed in oder to be edited (otherwise socket editing fails)
```
Comment (cdimauro) [inline src/domain/configure_ftp_cameras.py:109]: in order to

File: src/domain/ftps_server.py [hunk 3]
```diff
@@ -37,7 +37,12 @@ def on_logout(self, username):
 
     def on_file_received(self, file):
         logger.info("Received %s file from FTPS Camera.", file)
-        img_queue.put({"img": file, "img_id": f"ftp_camera_id"}) #TODO: fix camera id
+        img_queue.put({"img": file, "img_id": os.path.basename(os.path.dirname(file))})
+
+    def on_incomplete_file_received(self, file):
+        # remove partially uploaded files
```
Comment (cdimauro) [inline src/domain/ftps_server.py:43]: Redundant

File: src/domain/mainwindow.py [hunk 4]
```diff
@@ -241,7 +250,7 @@ def update_info_widget(self):
                 str(self.operation_mode.last_classified_animals))
 
     def url_input_dialog(self):
-        """Method to run dialog for insertion of an URL to fetch image from."""
+        """Method to run dialog for insertion of a URL to fetch image from."""
```
Comment (cdimauro) [inline src/domain/mainwindow.py:253]: I think that "an" is correct, instead of "a"

File: src/domain/animal_detection_mode.py [hunk 5]
```diff
@@ -41,7 +41,7 @@ def run(self):
                     self.camera_thread.append(camera.run())
                 elif camera.type == Camera.CameraTypes.FTPCamera and FTPsServer.ftps_server and not self.ftp_thread:
                     logger.info("Instantiating FTPS server...")
-                    FTPsServer.ftps_server.run()
+                    self.ftp_thread = FTPsServer.ftps_server.run()
```
Comment (cdimauro) [inline src/domain/animal_detection_mode.py:44]: Do you define / initialized self.ftp_thread somewhere in the class (__init__)?

File: src/domain/mainwindow.py (hunk 4 above)
Comment (stefanodellosa-personal) [reply src/domain/mainwindow.py:253]: I got educated that what matter is the sound of the initial letter. if it sounds like vowel then 'a' applies... Pycharm suggestion :)

File: src/domain/animal_detection_mode.py (hunk 5 above)
Comment (stefanodellosa-personal) [reply src/domain/animal_detection_mode.py:44]: As this is a specialization class of OperationMode class, self.ftp_thread = None is performed in OperationMode init()

File: src/domain/configure_ftp_cameras.py (hunk 2 above)
Comment (stefanodellosa-personal) [reply src/domain/configure_ftp_cameras.py:109]: i see in order to..

Comment (cdimauro) [reply src/domain/configure_ftp_cameras.py:109]: There's: "oder".

**Pull request merged**